test/unit/alignment/pairwise/fixture/local_affine_unbanded.hpp

```diff
@@ -41,12 +45,13 @@ static auto dna4_01 = []()
         // GTCTA
         "AACCGGTTTAACCGGTT"_dna4,
         "ACGTCTACGTA"_dna4,
-        align_config | align_cfg::scoring{nucleotide_scoring_scheme{match_score{4}, mismatch_score{-5}}},
+        align_config | seqan3::align_cfg::scoring{seqan3::nucleotide_scoring_scheme{match_score{4},
```

Comment:
What about `match_score` and `mismatch_score`, don't they need a `seqan3::` prefix too?